Dockerfile

```diff
@@ -6,6 +6,8 @@ FROM guacamole/guacd:1.5.4
 USER root
 RUN apk update && apk add openjdk11-jre-headless bash && apk cache clean
 COPY --from=BUILDER /src/target/guacamole-jetty-1.0.0-jar-with-dependencies.jar /guacamole-jetty.jar
+COPY config.properties /guacamole/config.properties
+RUN chmod 666 /guacamole/config.properties
```

Comment:
The path and filename are inconsistent from the code, please fix. Also make sure `/etc/guacamole` is owned by user `guacd`, and do not use `chmod` to allow anyone to write it.

Reply:
Actually a better solution is only create the `/etc/guacamole` folder (to make sure the correct ownership of the folder), without a default config file. Or even better, create the folder with correct ownership from `start.sh` - as this is more robust.